framework/validators/CompareValidator.php

```diff
@@ -85,28 +85,28 @@ public function init()
         if ($this->message === null) {
             switch ($this->operator) {
                 case '==':
-                    $this->message = Yii::t('yii', '{attribute} must be repeated exactly.');
+                    $this->message = Yii::t('yii', '{attribute} must be equal to "{compareInternal}".');
```

Comment:
I don't like how it's called now. It will be passed to translators and it's not clear what it is from the name. Also the word "internal" could be confusing.